src/aks-preview/HISTORY.rst

```diff
@@ -25,6 +25,8 @@ Pending
 0.5.162
 +++++++
 * Replace Image Cleaner related functions with stable version.
+* Add option `--enable-azure-container-storage` and supporting options `--storage-pool-name`, `--storage-pool-type`, `--storage-pool-sku`, `--storage-pool-size` for `az aks create` and `az aks update`. `az aks update` also supports `--azure-container-storage-nodepools` option.
```

Comment:
Please create a new section with a new version number and put the history info beneath. Also update version in `setup.py`.